core/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1,2 @@
+junit.jupiter.execution.parallel.enabled=true
+junit.jupiter.execution.parallel.mode.default=same_thread
```

Comment:
Why do we need to add these property files to several modules?

Reply:
The concurrent execution is still an experimental feature so it has to be enabled
https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-parallel-execution

Reply:
right, but I mean:
1) do we need it for _unit_ tests?
2) why not set those properties only in `testing`?